### PR TITLE
[System Properties] LPS-158206 Add new functionality to SystemProperties: get system properties by prefix(this function is needed when moving module framwork properties)

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I add a new method  and unit test in this pr. 
LPS-158206 depends on LPS-158153

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:acceptance

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:sf

**Comment by W2jx** [comment]
Hi @zhangyan0701 , Could you please help me scan test results when you free ? Thanks!

**Comment by W2jx** [comment]
ci:test:acceptance

File: portal-kernel/test/unit/com/liferay/portal/kernel/util/SystemPropertiesTest.java [hunk 1]
```diff
@@ -58,6 +61,31 @@ public void testGetProperties() {
...
+		Map<String, String> propertiesWithoutPrefix =
+			SystemProperties.getProperties(prefix, true);
+
+		Assert.assertTrue(propertiesWithoutPrefix.size() == 1);
+		Assert.assertEquals(
+			_TEST_VALUE, propertiesWithoutPrefix.get("test.key"));
+
+		propertiesWithPrefix = SystemProperties.getProperties(prefix, false);
+
+		Assert.assertTrue(propertiesWithPrefix.size() == 1);
```
Comment (tinatian) [inline portal-kernel/test/unit/com/liferay/portal/kernel/util/SystemPropertiesTest.java:85]: @W2jx Please find a better way to do this.

File: portal-kernel/test/unit/com/liferay/portal/kernel/util/SystemPropertiesTest.java [hunk 2]
```diff
@@ -58,6 +61,31 @@ public void testGetProperties() {
...
+	public void testGetPropertiesWithPrefix() {
+		String prefix =
+			SystemPropertiesTest.class.getName() + StringPool.PERIOD;
+
+		Map<String, String> propertiesWithPrefix =
+			SystemProperties.getProperties(prefix, false);
+
+		Assert.assertTrue(propertiesWithPrefix.isEmpty());
+
+		SystemProperties.set(_TEST_KEY, _TEST_VALUE);
```
Comment (tinatian) [inline portal-kernel/test/unit/com/liferay/portal/kernel/util/SystemPropertiesTest.java:74]: @W2jx It is not that obvious the _TEST_KEY's prefix is `SystemPropertiesTest.class.getName() + StringPool.PERIOD`

**Comment by tinatian** [comment]
@W2jx Please see my comment and resend.

Thanks.
Tina

**Pull request closed without merging**